tm2/pkg/std/memfile.go

```diff
@@ -98,3 +108,7 @@ func SplitFilepath(filepath string) (dirpath string, filename string) {
 		return strings.Join(parts, "/"), ""
 	}
 }
+
+func isValidLength(pkgPath string, limitation int) bool {
+	return len(pkgPath) <= limitation
+}
```

Comment:
There is no need for this helper, you can do this conditional check in the moment of calling:
```go
if len(pkgPath) > limitation {
    return fmt.Errorf...
}
```

Reply:
Done.